partner_email_check/models/res_partner.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2019 Komit <https://komit-consulting.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import _, api, models
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools import config
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from email_validator import (
+        validate_email,
+        EmailSyntaxError,
+        EmailUndeliverableError,
+    )
+except ImportError:
+    _logger.debug('Cannot import "email_validator".')
+
+    validate_email = None
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def copy_data(self, default=None):
+        res = super(ResPartner, self).copy_data(default=default)
+        if self._should_filter_duplicates():
+            for copy_vals in res:
+                if "email" in copy_vals:
+                    del copy_vals["email"]
```

Comment:
not a blocker but you could do:
```suggestion
            for copy_vals in res:
                copy_vals.pop("email", None)
```